### ISSUE TITLE
[Feature Request] Option to play vanilla OoT with all the QoL changes from Randomizer

### ISSUE THREAD

**Issue by ShadowOne333** [TestRunnerSRL/OoT-Randomizer#999]
Good day.

I've been following the OoT Randomizer for a while, and have done a couple of Rando playthroughs with it. It's such a refreshing thing to experience.

However, one thing I'd like is for being able to use all of the Quality of Life hacks that the Randomizer has gathered over the years, and implement over vanilla Ocarina of Time.
Things such as the faster speed, D-Pad boots+Ocarina, Bunny Hood speeds up Link (like in MM), bugfixes and a lot of other stuff, alongside the default perk of using v1.0 as the base, having the game uncensored thanks to that as well.

So with that, I'd like to suggest having such an option possible from the Randomizer settings.
Perhaps having it as a checkbox that only enables all of the ASM and QoL hacks into vanilla Ocarina, and still leave vanilla OoT with all its cutscenes and other stuff. (Also, I'm not sure if this is already possible in Rando or not, but having something like Master Quest dungeons/Title screen as an option could be neat too).

Basically, just original OoT but with every QoL change from Rando into it.

Could something like this be possible?

PS: I know BetterOoT exists, but that project is quite obsolete (1+ years inactive) compared to current Rando standards, and also that project completely removes cutscenes and other changes which the author has already rejected to restore back from the original game.

**Comment by IAmNamedGravy** [comment]
https://ootrandomizer.com/seed/vanilla

**Comment by flagrama** [comment]
We are the ones who removed cutscenes and reject restoring them. Better OoT lets you choose them.

**Comment by ShadowOne333** [comment]
> https://ootrandomizer.com/seed/vanilla

I was aware of that seed. However, there are still a lot of things that stray away from vanilla OoT with it, these are just a few at the very beginning:

- New game begins with the Light Medallion
- Cutscenes seem to be absent (Intro cutscene, Saria cutscene, and so on)
- Title Screen still says "Randomizer" (though this is expected)
- Randomizer seed displays on the File Select screen
- Map on New Game begins with other areas already marked as plot points

There are still a lot of other things, but I don't have the time to go through them all.

> We are the ones who removed cutscenes and reject restoring them. Better OoT lets you choose them I was pretty sure. Bunny Hood is realistically the only thing we have that BOoT doesn't yet have.

Iirc, there are still a lot of cutscenes still missing. Even the most minor ones are, which is not desired when going for a full original Ocarina of Time experience.

**Comment by flagrama** [comment]
Well that's out of scope for Randomizer. Maybe Fig will do something with the decompiled source once they are done working on that. Otherwise someone else will have to do it in their own personal fork.

**Comment by ShadowOne333** [comment]
I am willing to make a fork on my own and try to make this work.
I located some of the code that seems to remove cutscenes at both `cutscenes.asm` and also `Patches.py`, alongside a lot of what seems to be save overwrites in the [Patches](https://github.com/ShadowOne333/OoT-Randomizer/blob/Dev/Patches.py#L924) file, which could potentially restore a bunch of the minor cutscenes. I will do some more research about this.

For the title screen, I believe there's also a hack that changes it to the Rando one in the [Patches.py](https://github.com/ShadowOne333/OoT-Randomizer/blob/Dev/Patches.py#L32) file.

For the hash appearing on the File Select screen, I'm not sure yet, but maybe NOP'ing the instruction that calls for `draw_file_select_hash` inside the [hacks.asm](https://github.com/AmazingAmpharos/OoT-Randomizer/blob/6d7813ca9950795e48af6978998d7a1ba14cba67/ASM/src/hacks.asm#L182) file?

I'm still  unsure how many things I might be missing, but if anyone has any suggestions or so, please let me know.

**Comment by TestRunnerSRL** [comment]
This is definitely outside the scope of this project. I'm also confused since you want some QoL feature but not others. So I'm not exactly sure what you goal is. If you want to run your own custom set, you're welcome to make a fork. The vanilla seed is just using plando to force vanilla items at the vanilla locations. If you're asking for help on your fork, this is not the place to do so.

**Comment by ShadowOne333** [comment]
> I'm also confused since you want some QoL feature but not others. So I'm not exactly sure what you goal is. 

Sorry, I meant all of the QoL that Rando uses for OoT. Unless I somehow mentioned that I wouldn't include one specifically? What part confused you about some QoL but not others?
